boards/arm/thingy91x_nrf9161/thingy91x_nrf9161_ns.dts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/dts-v1/;
+#include <nordic/nrf9161ns_laca.dtsi>
+#include "thingy91x_nrf9161_common.dts"
+
+/ {
+	chosen {
+		zephyr,flash = &flash0;
+		zephyr,sram = &sram0_ns;
+		zephyr,code-partition = &slot0_ns_partition;
+	};
+};
+
+/* Disable Zephyr driver for UART1. The nrfx driver is used for interfacing with UART1
```

Comment:
This comment is outdated. The UART trace backend now uses the Zephyr driver depending on the device tree chosen node for modem traces:

```
	chosen {
		nordic,modem-trace-uart = &uart1;
	};
```

So it depends on if we want modem traces to be enabled by default or not. It might make sense to disable by default for battery usage, and use a snippet to enable when needed. But perhaps set the default directly in thingy91x_nrf9161_common.dts? Or does it need to differ between secure/non secure?

Reply:
changed to comment to make clear why this is disabled by default in non-secure.